inc/search.class.php

```diff
@@ -3326,9 +3468,11 @@ static function addDefaultSelect($itemtype) {
     * @param integer $meta_type    meta type table ID (default 0)
     *
     * @return string Select string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addSelect($itemtype, $ID, $meta = 0, $meta_type = 0) {
-      global $DB, $CFG_GLPI;
+   public function addSelect($itemtype, $ID, $meta = 0, $meta_type = 0) {
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.